hip-config.cmake.in

```diff
@@ -60,9 +63,11 @@ if(HIP_COMPILER STREQUAL "clang")
     else()
       set(HIP_CLANG_ROOT /opt/rocm/llvm)
     endif()
-  else()
+  elseif (HIP_CXX_COMPILER MATCHES ".*clang++")
```

Comment:
This regex is not correct. Can we just use the `else` instead?

Reply:
`.*clang\\+\\+`